### ISSUE TITLE
[Android] Mute my application's sound when another application sound is open

### ISSUE THREAD

**Issue by hoangsang0109** [TheWidlarzGroup/react-native-video#2813]
Hello, thanks for your library.

I have an issue like this:

When I play music on my app then I play music on another app (For example Youtube).

The result is 2 sounds playing together. How to mute my application's sound when another application's sound is open?

## Platform

Which player are you experiencing the problem on:
* Android

**Comment by freeboub** [comment]
Do you use playInBackground ?
do you use disableFocus ?

**Comment by hoangsang0109** [comment]
> Do you use playInBackground ? do you use disableFocus ?

Yes, I use both. This is my code, please check. Thanks

`      <Video
        ref={videoRef}
        volume={1}
        repeat={repeat}
        audioOnly={true}
        source={{ uri: audioSource }}
        mixWithOthers={undefined}
        paused={isPaused}
        onLoad={onLoadVideo}
        onEnd={onEnd}
        onProgress={onProgress}
        ignoreSilentSwitch={'ignore'}
        playWhenInactive={true}
        playInBackground={true}
        disableFocus={true}
      />`

**Comment by freeboub** [comment]
Did you try to put disableFocus to false ? I think it should solve your issue

**Comment by jaswant25** [comment]
> Did you try to put disableFocus to false ? I think it should solve your issue

Yes , I am facing the same issue, It doesn't pause other audio streams.

**Comment by jeandiasl** [comment]
I am facing the same issue, It doesn't pause other audio streams.
